package.json

```diff
@@ -48,10 +25,11 @@
     "jest-preset.js"
   ],
   "scripts": {
-    "test": "jest && npm run test:git-history",
-    "test:git-history": "commitlint --from origin/master --to HEAD",
+    "test": "npm run test:unit",
```

Comment:
`npm run test:lint` is missing

Reply:
Ah yes, I need to remove it from the tests since linting does not exist in this repo